frontend/src/components/ProductCard/StyledProductCard.ts

```diff
@@ -13,17 +13,19 @@ export const StyledProductCard = styled.div`
   background-color: #fff;
   border: 1px solid #e2e6e9;
   border-radius: 8px;
-  width: 272px;
+  width: 100%;
+  max-width: 288px;
   height: 440px;
   padding-bottom: 32px;
 
-  @media (min-width: 768px) {
+  @media (min-width: 640px) and (max-width: 1199px) {
```

Comment:
If you have another media query min-width 1200px you don't need this max-width 1199px.